global.js

```diff
@@ -103,6 +103,8 @@ function onTextTyped (key) {
         }
       } else if (link.link.tagName === 'BUTTON') {
         link.link.click()
+      } else if (isCurrentlyInInput()) {
+        link.link.focus();
```

Comment:
Input elements like checkboxes and radialboxes do not change using this method. Can you add a check for such elements and simulate a `click` on them instead to toggle them.

Reply:
I've checks for input types (checkbox/radio) and also for select elements so that you can move through the options.